src/main/java/core/basesyntax/util/filewriter/FileWriter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.util.filewriter;
+
+public interface FileWriter {
+    void write(String resultingReport, String s);
```

Comment:
The parameter names in the interface method signature should be descriptive. The current names 'resultingReport' and 's' do not clearly describe what they represent. Consider renaming 's' to 'filePath' to indicate that it should be the path to the file where the report will be written.